src/execution_plan/ops/op_index_scan.c

```diff
@@ -6,42 +6,40 @@
 
 #include "op_index_scan.h"
 
-int IndexScanToString(const OpBase *ctx, char *buff, uint buff_len) {
+/* Forward declarations. */
+static Record IndexScanConsume(OpBase *opBase);
+static OpResult IndexScanReset(OpBase *opBase);
+static void IndexScanFree(OpBase *opBase);
+
+static int ToString(const OpBase *ctx, char *buff, uint buff_len) {
 	const IndexScan *op = (const IndexScan *)ctx;
 	int offset = snprintf(buff, buff_len, "%s | ", op->op.name);
 	offset += QGNode_ToString(op->n, buff + offset, buff_len - offset);
 	return offset;
 }
 
-OpBase *NewIndexScanOp(Graph *g, QGNode *n, uint node_idx, RSIndex *idx, RSResultsIterator *iter) {
-	IndexScan *indexScan = malloc(sizeof(IndexScan));
-	indexScan->g = g;
-	indexScan->n = n;
-	indexScan->idx = idx;
-	indexScan->iter = iter;
-	indexScan->nodeRecIdx = node_idx;
+OpBase *NewIndexScanOp(const ExecutionPlan *plan, Graph *g, const QGNode *n, RSIndex *idx,
+					   RSResultsIterator *iter) {
```

Comment:
Indentation?

Reply:
Just Github formatting weirdness (this applies for all your indentation comments on this PR). Github prints tabs as 8 spaces rather than 4; in our editors arguments on the second line are aligned with arguments on the first.